src/glmfit.jl

```diff
@@ -205,6 +207,19 @@ function updateμ!(r::GlmResp{V,D,L}) where {V<:FPVector,D<:NegativeBinomial,L<:
     end
 end
 
+function updateμ!(r::GlmResp{V,D,L}) where {V<:FPVector,D,L<:PowerLink}
```

Comment:
Isn't this method identical to the fallback method, except for the `L(pl.λ)` part? If so it seems that using `r.link` instead would work in all cases and avoid duplication.

Reply:
I agree.

Reply:
Remove this updateμ! function and updated the general updateμ! function.